lightning_pose_app/ui/extract_frames.py

```diff
@@ -422,6 +424,14 @@ def _render_streamlit_fn(state: AppState):
     # initialize the file uploader
     uploaded_files = st.file_uploader("Select video files", accept_multiple_files=True)
 
+    st.caption("*For files exceeding 200MB, please utilize the Lightning Pose API for seamless upload")
```

Comment:
let's remove this for now, until we have clearer instructions for how to upload videos through the Lightning API

Reply:
Done